### ISSUE TITLE
SSL certificate expired

### ISSUE THREAD

**Issue by sebsto** [aws-amplify/amplify-console#1862]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-console/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-console/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-console/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### App Id

https://eu-central-1.console.aws.amazon.com/amplify/home?region=eu-central-1#/d3go33mcbbvvyj/

### Region

eu-central-1

### Amplify Console feature

Not applicable

### Describe the bug

App is deployed since 2019.
SSL Certificate did not renew and expired on May 18 2021
No changes made on DNS / domain settings in months.

```
curl -v https://nata.coach
*   Trying 54.192.66.93...
* TCP_NODELAY set
* Connected to nata.coach (54.192.66.93) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/cert.pem
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (OUT), TLS alert, certificate expired (557):
* SSL certificate problem: certificate has expired
* Closing connection 0
curl: (60) SSL certificate problem: certificate has expired
```

### Expected behavior

I expect the SSL certificate to renew automatically.

### Reproduction steps

open a browser, navigate to https://nata.coach

### Build Settings

```yaml
n/a
```

### Additional information

I found similar issues :  https://github.com/aws-amplify/amplify-console/issues/1327 https://github.com/aws-amplify/amplify-console/issues/996

**Comment by sebsto** [comment]
I will apply the workaround described [here](https://github.com/aws-amplify/amplify-console/issues/996) (delete the custom domain and re-create one) because I cannot afford to keep that website unaccessible for longer.

But I would like to understand the root cause WHY the certificate was not auto-renewed.

I did not touch the DNS domain and this record is present in my DNS
`_5ca925bd11fd1b1931f124916a577a0f.nata.coach CNAME _96fc9623097310fe1fd9e92516c6165d.hkvuiqjoua.acm-validations.aws. `

**Comment by sebsto** [comment]
impact mitigated - app is back online.
Keeping this issue open for root cause analysis

**Comment by sebsto** [comment]
When I first discovered this yesterday I through about an issue with Amplify Console. But after doing a deeper analysis during the day, I arrived at the conclusion that I probably deleted a CNAME record from Route53.

I should have had TWO CNAME record on that domain (because I have two certificates, one being used by another Cloudfront distribution I do manage manually)

So the only possible explanation is that I “cleaned up” CNAME a bit too aggressively during the past year and I don’t remember it.

User error - I am closing this issue.  Thanks for having checked.
